### PR TITLE
[terra-framework] Removed inline style for dev-site components Part 6

### LINKED ISSUE #657: Remove inline styles in dev site components
# Feature Request

## Description

We want to remove inline styles from our dev site components to set a better example for consumers of terra.  As a replacement, we should pass in classes and use css to style the components.

### PR DESCRIPTION
**Pull request by pranav300**
### Summary
Removed all inline styles and replaced the same styles with external CSS for the following packages.

1. HookShot
2. Navigation Layout

Partially resolves #657

### CODE REVIEW

File: packages/terra-navigation-layout/src/terra-dev-site/doc/example/NavigationLayoutDocCommon.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  .content-wrapper2 {
+    color: #808080;
+    left: 50%;
+    position: absolute;
+    top: 50%;
+    transform: translate3d(-50%, -50%, 0);
+  }
+
+  button {
```
Comment (bjankord) [inline packages/terra-navigation-layout/src/terra-dev-site/doc/example/NavigationLayoutDocCommon.scss]: lets change this to a class, `.button`

**Comment by pranav300** [comment]
I will be creating a new PR due to some build issues.

File: packages/terra-hookshot/src/terra-dev-site/doc/example/HookshotDocCommon.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  .select-wrapper {
+    height: 30px;
+    width: 200px;
+  }
+
+  .heading-wrapper {
+    height: 25px;
+  }
+
+  label {
```
[the remark this answers is not in the document]
Comment (pranav300) [reply packages/terra-hookshot/src/terra-dev-site/doc/example/HookshotDocCommon.scss]: Fixed here https://github.com/cerner/terra-framework/pull/742/commits/a15a90709635d11fcccf4a0e031c02968a5cd8c2.

File: packages/terra-navigation-layout/src/terra-dev-site/doc/example/NavigationLayoutDocCommon.scss (hunk 1 above)
Comment (pranav300) [reply packages/terra-navigation-layout/src/terra-dev-site/doc/example/NavigationLayoutDocCommon.scss]: Fixed here https://github.com/cerner/terra-framework/pull/742/commits/8262dbdbd969709d4d0a1cc42ba3188a1269b82c.

**Pull request closed without merging**